src/Core.Nhibernate/Profiles/IDbProfileConfig.cs

```diff
@@ -0,0 +1,9 @@
+﻿using AutoMapper;
+
+namespace IdentityServer3.Core.Models
+{
+    public interface IDbProfileConfig
+    {
+        Profile GetProfile();
```

Comment:
I don't like the naming of this. It's extremely non-obvious that it has anything to do with AutoMapper.

Reply:
Maybe, `IMappingProfileProvider`?

Reply:
Have now removed as no longer required with revised approach.